app/src/main/java/com/example/munchbox/ui/RestaurantHubScreen.kt

```diff
@@ -56,13 +57,23 @@ fun RestaurantHubScreen(
             selectedOptions.value = setOf<DietaryOption>()
         }
     }
+    val cancelMealCOROUTINE: (meal : Meal) -> Unit = {
+        coroutineScope.launch {
+            val mealCancelReturn = storageService.restaurantService().updateMeal(restaurant.restaurantID, it.mealID, null, null, Date())
```

Comment:
does `it` reference the `meal` you passed in? i literally didn't know that was a feature

Reply:
lol yeah it's the "implicit parameter"